### PR TITLE
v1.22 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
 
+## 1.22.0 Release
+- Release Date: August 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
```
Comment (amyblais) [inline CHANGELOG.md]: These need some more details.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+ - Changed "about" section references to use the site name when it is configured in **System Console > Custom Branding > Site Name**.
+ - Added support for plus-sign and period/dot in custom URL schemes.
+ - Added "Edit profile" button to right-hand side menu and to users' own profile pop-over.
+ - Message draft is now saved when closing the app.
+ - Removing a link preview on webapp now also removes it on the mobile app.
+ - Added ability to select and copy channel header text.
+
+### Bug Fixes
+ - Fixed a few mobile app crash / fatal error issues.
+ - Fixed an issue where old theme's colors were applied to navigation bar during screen transitions.
```
Comment (enahum) [inline CHANGELOG.md]: remove

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,45 @@
...
+ - Fixed an issue on iOS where returning from landscape video playback moved status bar under notch.
```
Comment (enahum) [inline CHANGELOG.md]: this one was done a few releases ago

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,45 @@
...
+#### Android Q Improvements
+
+### Improvements
+ - Added a setting for tablets to enable or disable fixed sidebar.
+ - Changed "about" section references to use the site name when it is configured in **System Console > Custom Branding > Site Name**.
+ - Added support for plus-sign and period/dot in custom URL schemes.
+ - Added "Edit profile" button to right-hand side menu and to users' own profile pop-over.
+ - Message draft is now saved when closing the app.
+ - Removing a link preview on webapp now also removes it on the mobile app.
+ - Added ability to select and copy channel header text.
```
Comment (enahum) [inline CHANGELOG.md]: and purpose

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,46 @@
...
+- Release Date: August 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
+
+#### React Native Navigation
```
Comment (esethna) [inline CHANGELOG.md]: Doesn't need a highlight, this is a backend change

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,46 @@
...
+
+#### Android Q Improvements
```
Comment (esethna) [inline CHANGELOG.md]: Support for iOS13 and Android Q

Added support for iOS13 and Android Q to be released later this year.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,46 @@
...
+ - Added ability to select and copy channel header text and purpose.
+
+### Bug Fixes
+ - Fixed a few mobile app crash / fatal error issues.
+ - Fixed an issue where timestamps were off on Android.
+ - Fixed an issue where contents of ephemeral posts from /giphy were not being displayed on mobile.
+ - Fixed an issue where team/channel page dots at the bottom of left-hand side overlapped with the last Direct Message channel.
+ - Fixed an issue where network reconnection incorrectly showed refreshing messages failed.
+ - Fixed an issue with a dark blue section at the bottom of channel list on iPhone X.
+ - Fixed an issue where "Message failed to send" had confusing app badge behaviour.
```
Comment (esethna) [inline CHANGELOG.md]: confusing > incorrect

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,46 @@
...
+ - Removing a link preview on webapp now also removes it on the mobile app.
+ - Added ability to select and copy channel header text and purpose.
+
+### Bug Fixes
+ - Fixed a few mobile app crash / fatal error issues.
+ - Fixed an issue where timestamps were off on Android.
+ - Fixed an issue where contents of ephemeral posts from /giphy were not being displayed on mobile.
+ - Fixed an issue where team/channel page dots at the bottom of left-hand side overlapped with the last Direct Message channel.
+ - Fixed an issue where network reconnection incorrectly showed refreshing messages failed.
+ - Fixed an issue with a dark blue section at the bottom of channel list on iPhone X.
```
Comment (esethna) [inline CHANGELOG.md]: Fixed an issue with the channel sidebar theme colors not being respected on iPhone X

**Comment by amyblais** [comment]
@enahum Can you help merge this PR?

**Pull request merged**